manimlib/mobject/probability.py

```diff
@@ -182,6 +186,17 @@ def add_axes(self):
             ticks.add(tick)
         y_axis.add(ticks)
 
+        if self.include_x_ticks == True:
+            ticks1 = VGroup()
+            widths = np.linspace(0, self.width, self.n_ticks_x + 1)
+            label_values = np.linspace(0, len(self.bar_names), self.n_ticks_x + 1)
+            for x, value in zip(widths, label_values):
+                tick1 = Line(UP*0.05, DOWN*0.05)
+                tick1.set_width(self.tick_width_x)
```

Comment:
Ticks on the y axis can use `set_width` because they are horizontal.
But here, `set_width` won't make sense.
And why you named this local variable to `tick1`? `tick` is also available.
These two lines should be modified to:
```py
tick = Line(DOWN, UP)
tick.set_height(self.tick_width_x) # maybe call it `tick_height` or `tick_length_x` is better?
```

Reply:
They're mainly because I was confused at what each line does at first:)